core/src/test/scala/com/nvidia/spark/rapids/tool/tuning/QualificationAutoTunerSuite.scala

```diff
@@ -719,7 +719,7 @@ class QualificationAutoTunerSuite extends BaseAutoTunerSuite {
           |--conf spark.plugins=com.nvidia.spark.SQLPlugin
           |--conf spark.rapids.memory.host.offHeapLimit.enabled=true
           |--conf spark.rapids.memory.host.offHeapLimit.size=80g
-          |--conf spark.rapids.memory.pinnedPool.size=48640m
+          |--conf spark.rapids.memory.pinnedPool.size=40g
```

Comment:
Also, when I run this UT, these are the values returned from `calcOverallMemory()` -



<img width="1093" height="401" alt="image" src="https://github.com/user-attachments/assets/34187e1b-5867-430a-9f58-9c64b96139e0" />


I noticed that `pinnedMem` + `spillMem` > `executorMemOverhead`. Is this expected (because `hostOffHeapLimitSizeMB` will handle this?

Reply:
The hostOffHeapLimitSizeMB will be the value user enforced or memoryOverhead + offHeap.
And the default PINNED_MEM_OFFHEAP_RATIO is 50%. If we don't set enforced offHeapLimit.size, pinnedMem will be 50% of memoryOverhead + offHeap. Spilll will be the same as pinnedMemory.
But if the enforced offHeapLimit.size is more than memoryOverhead + offHeap, it may cause the pinnedMem + spillMem > executorMemOverhead + offHeap.
Do we need to check the overall memory exceeds or not?

Reply:
Follow up on #1880